### PR TITLE
add docstrings

### LINKED ISSUE #153: The ABCParticles do not have docstrings

### PR DESCRIPTION
**Pull request by itziakos**
fixes #153

### CODE REVIEW

File: simphony/cuds/abstractparticles.py [hunk 1]
```diff
@@ -17,48 +17,346 @@ class ABCParticles(object):
...
+            The uid of the added particle.
+
+        Raises
+        ------
+        ValueError :
+            when the new particle already exists in the container.
+
+        Examples
+        --------
+        Having a Particle and a ParticleContainer just call the function
```
Comment (nathanfranklin) [inline simphony/cuds/abstractparticles.py]: `ParticleContainer`.   Should it be `Particles` or `particle container`?

(same question applies to docstring for `add_bond`)

File: simphony/cuds/abstractparticles.py [hunk 2]
```diff
@@ -17,48 +17,346 @@ class ABCParticles(object):
...
+        ------
+        ValueError :
+            When the new particle already exists in the container.
+
+        Examples
+        --------
+        Having a Bond and a ParticleContainer, just call the function
+        passing the Bond as parameter.
+
+        >>> bond = Bond()
```
Comment (nathanfranklin) [inline simphony/cuds/abstractparticles.py]: Bond() is missing `particles` parameter.

should it be something like
```
bond =  Bond(particles=(particle1.uid, particle2.uid))

```

**Comment by nathanfranklin** [comment]
I had just two comments but otherwise, it looks good to me.

File: simphony/cuds/abstractparticles.py (hunk 1 above)
Comment (itziakos) [inline simphony/cuds/abstractparticles.py]: Good point, I will fix that

**Comment by itziakos** [comment]
PR updated

**Comment by nathanfranklin** [comment]
:+1:

**Pull request merged**